### PR TITLE
Add Prometheus rules for multicluster deployment

### PR DESCRIPTION
**Pull request by VerosK**
I've added some Prometheus rules to get multicluster graphs running.

### CODE REVIEW

File: rules/apps.libsonnet [hunk 1]
```diff
@@ -21,9 +21,9 @@
             // quantile_over_time(...) which would otherwise not be possible.
             record: 'node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate',
             expr: |||
-              sum by (namespace, pod, container) (
+              sum by (cluster, namespace, pod, container) (
```
Comment (brancz) [inline rules/apps.libsonnet]: We can't assume this label, but we have configuration for it in the `$._config.clusterLabel` variable which can be used here.

**Comment by VerosK** [comment]
Thank you for pointing out. 

I've added handling of `clusterLabel` to the right places.  

Should I squeeze the commits into one and open new PR?

File: rules/apps.libsonnet (hunk 1 above)
Comment (VerosK) [reply rules/apps.libsonnet]: Thank you for pointing out.

Next commit is using the configuration.

**Comment by brancz** [comment]
lgtm

**Pull request merged**